File: Array/TheMinimumNumberOfMoves .cpp
#include<iostream>
using namespace std;

int main()
{
    int step,test,n,sum,wage;

    cin>>test;

    while(test--)
    {
        sum=0;
        cin>>n;
        int minimum=10000;
        for(int i=0; i<n; i++)
        {
            cin>>wage;
            if(minimum>wage)
                minimum = wage;

            sum+=wage;
        }

        step = sum - (n*minimum);

        cout<<step<<endl;

    }

}

File: Array/TheMinimumNumberOfMoves .py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Thu Dec 17 14:11:02 2020

@author: gourob
"""

test = int(input())

for t in range(test):
    
    n=int(input()) 
    w= input()
    wage=[int(i) for i in w.split()]
    
    step= sum(wage) - (n* min(wage))
    print(step)
    